crates/bevy_render/src/texture/ktx2.rs

```diff
@@ -1363,100 +1242,184 @@ pub fn ktx2_format_to_texture_format(
         ktx2::Format::EAC_R11G11_SNORM_BLOCK => TextureFormat::EacRg11Snorm,
         ktx2::Format::ASTC_4x4_UNORM_BLOCK | ktx2::Format::ASTC_4x4_SRGB_BLOCK => {
```

Comment:
We could also map from these ktx2 formats to the `TextureFormat::Astc { block, .. }` member somehow... I don't know if the code would be cleaner. Just a thought and I won't block (HA!) on it.